### PR TITLE
Redux and redux-saga

### PR DESCRIPTION
**Pull request by P-Michalski**
Added basic application state, using redux and redux-saga. It consists of simple actions, that will be later implemented into the app (we will soon add more as we will need them). For now, 4 lines in saga are commented, as I am not yet provided with function  names for getting necessary data from API and thats why console shows that some things are declared but not used. We will change that as soon as I get that info.

### CODE REVIEW

File: src/features/moviesSlice.js [hunk 1]
```diff
@@ -21,14 +21,14 @@ const moviesSlice = createSlice({
...
-        fetchPopularMovies: state => {
+        fetchPopular: state => {
             state.status = "loading";
         },
         setMoviesData: (state, { payload: data }) => {
             state.moviesData = data;
-            state.status = undefined;
+            state.status = "success";
         },
-        fetchPopularMoviesError: state => {
```
Comment (waltad) [inline src/features/moviesSlice.js]: In 31 code line name was changed with fetchPopularMoviesError to fetchPopulaError. In 47 code line is declare again fetchPopularMoviesError.

File: src/features/moviesSlice.js [hunk 2]
```diff
@@ -21,14 +21,14 @@ const moviesSlice = createSlice({
         setLastPage: (state, { payload: totalPages }) => {
             state.page = totalPages;
         },
-        fetchPopularMovies: state => {
+        fetchPopular: state => {
```
Comment (waltad) [inline src/features/moviesSlice.js]: In 24 code line name was changed with fetchPopularMovies to fetchPopular. In 45 code line is declare again fetchPopularMovies.

File: src/features/moviesSlice.js (hunk 1 above)
Comment (P-Michalski) [reply src/features/moviesSlice.js]: Thanks, I fixed it now :D

File: src/features/moviesSlice.js (hunk 2 above)
Comment (P-Michalski) [reply src/features/moviesSlice.js]: Fixed ^-^

**Pull request merged**